src/js/pages/JobsPage.js

```diff
@@ -0,0 +1,66 @@
+import mixin from 'reactjs-mixin';
+import React from 'react';
+import {RouteHandler} from 'react-router';
+
+import Page from '../components/Page';
+import SidebarActions from '../events/SidebarActions';
+import TabsMixin from '../mixins/TabsMixin';
+
+class JobsPage extends mixin(TabsMixin) {
+  constructor() {
+    super();
```

Comment:
Can you explain why? I've seen inconsistent usage across the codebase and in this case I'd argue that TabsMixin has no constructor per se so I don't see the need for `...arguments` here?